service/src/main/java/bio/terra/profile/service/spendreporting/azure/model/mapper/SpendDataItemCategoryMapper.java

```diff
@@ -0,0 +1,20 @@
+package bio.terra.profile.service.spendreporting.azure.model.mapper;
+
+import bio.terra.profile.service.spendreporting.azure.model.SpendCategoryType;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SpendDataItemCategoryMapper {
+  public static final String AZURE_COMPUTE_RESOURCE_TYPE = "microsoft.compute";
+  public static final String AZURE_STORAGE_RESOURCE_TYPE = "microsoft.storage";
+
+  public SpendCategoryType mapResourceCategory(String resourceCategory) {
+    if (resourceCategory.startsWith(AZURE_COMPUTE_RESOURCE_TYPE)) {
```

Comment:
The storage and compute resource providers in Azure are surfaced as `Microsoft.Storage` and `Microsoft.Compute` respectively -- this code may want to defensively do a toLowerCase to guard against any inconsistencies from the API.